Xamarin.Forms.Platform.Tizen/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms.Shapes;
+using SkiaSharp;
+using FormsRectangle = Xamarin.Forms.Rectangle;
+
+namespace Xamarin.Forms.Platform.Tizen
+{
+    public static class GeometryExtensions
+    {
+        public static SKPath ToSKPath(this Geometry geometry)
+        {
+            SKPath path = new SKPath();
```

Comment:
This line can be removed. 
```suggestion
            SKPath path = new SKPath();
```